homeassistant/components/webostv/media_player.py

```diff
@@ -271,6 +272,10 @@ def media_image_url(self):
             icon = self._app_list[self._current_source_id]['largeIcon']
             if not icon.startswith('http'):
                 icon = self._app_list[self._current_source_id]['icon']
+            if self._last_icon and \
```

Comment:
I'd suggest adding a comment to explain this branch of code so someone looking at this in the future doesn't wonder what the heck this is for.  :)